src/BasicRum/Filters/Primary/AbstractFilter.php

```diff
@@ -10,7 +10,7 @@ abstract class AbstractFilter implements \App\BasicRum\Report\PrimaryFilterableI
 
     protected $searchValue;
 
-    public function __construct(string $condition, string $searchValue)
+    public function __construct(string $condition = null, string $searchValue = null)
```

Comment:
Hello @korzol 

Why do we change this function definition? We should never allow a filter to be initialized without condition and value? Did you get any PHP errors that forced you to do this?